src/Form/form.less

```diff
@@ -240,6 +253,37 @@ base/forms.less
         }
       }
     }
+
+    // Radio button specific styles
+    .form-element-radio-button {
+      input[type="radio"] {
+        & + .form-element-radio-button-decoy {
+          .icon-radio-button {
```

Comment:
#nit Let's get newlines above `.icon-radio-button, `& + .form-element-radio-button-decoy` and `input[type="radio"]`